File: app/src/main/java/com/example/dcct/model/internet/PostQueryEntity.java
package com.example.dcct.model.internet;

public class PostQueryEntity {

    /**
     * drugOne : 粉葛
     * drugTwo : 白石英
     */

    private String drugOne;
    private String drugTwo;
    private long uid;

    public PostQueryEntity(String drugOne, String drugTwo,long uid) {
        this.drugOne = drugOne;
        this.drugTwo = drugTwo;
        this.uid = uid;
    }

    @Override
    public String toString() {
        return "PostQueryEntity{" +
                "drugOne='" + drugOne + '\'' +
                ", drugTwo='" + drugTwo + '\'' +
                ", uid=" + uid +
                '}';
    }

}

File: app/src/main/java/com/example/dcct/bean/PostQueryEntity.kt
package com.example.dcct.bean

class PostQueryEntity(
        /**
         * drugOne : 粉葛
         * drugTwo : 白石英
         */
        private val drugOne: String, private val drugTwo: String, private val uid: Long) {
    override fun toString(): String {
        return "PostQueryEntity{" +
                "drugOne='" + drugOne + '\'' +
                ", drugTwo='" + drugTwo + '\'' +
                ", uid=" + uid +
                '}'
    }

}